docs/spinitron.json

```diff
@@ -753,6 +756,160 @@
           }
         }
       }
+    },
+    "/non-music-items": {
+      "get": {
+        "summary": "Returns non-music items optionally filtered by {start} and/or {end} datetimes",
+        "description": "Get non-music items optionally filtered by a datetime range. Only past non-music items will be returned.\n",
```

Comment:
add the word "public" to both summary and descriptions, i.e. "Returns public non-music items ..." to remind us to always filter out the non-public items